examples/unkey-cli/apps/nextjs/app/api/unkey/route.ts

```diff
@@ -0,0 +1,28 @@
+import { Unkey } from "@unkey/api";
+import { NextResponse } from "next/server";
+
+export async function POST(request: Request) {
+  const { id, redirect, code } = await request.json();
+  if (!process.env.UNKEY_ROOT_KEY || !process.env.UNKEY_API) {
```

Comment:
we used `UNKEY_API_ID` everywhere, let's keep that as it is more precise
```suggestion
  if (!process.env.UNKEY_ROOT_KEY || !process.env.UNKEY_API_ID) {
```